src/wallet/home_cards/manager.rs

```diff
@@ -110,6 +110,18 @@ impl HomeCardsManager {
             .await?;
         self.save_cards(updated_cards).await
     }
+
+    /// Clears the home cards from the `HomeCardsStorage`.
+    /// Notifies `HomeCardsObserver`.
+    #[uniffi::method]
+    pub async fn wallet_reset(&self) -> Result<()> {
```

Comment:
I would name it `wallet_resetted` to follow the past tense used for other actions (`wallet_created`, `link_received`, etc)

Reply:
I think `reset` is the correct past participle form. [Here](https://twominenglish.com/resetted-or-reset/#:~:text=Instead%2C%20it%20maintains%20the%20same,as%20its%20base%20form%3A%20reset.) is a reference.